### PR TITLE
EASY-2846: Add Rule 2.7 on original-filepaths.txt

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
Fixes EASY-2846

# Description of changes
* Add Rule 2.7 on original-filepaths.txt
* Add reference to 2.7.2 to Rule 3.2.4

# How to test

# Related PRs
* [ ] [easy-validate-dans-bag](https://github.com/DANS-KNAW/easy-validate-dans-bag/pull/108)

# Notify
@DANS-KNAW/easy

### CODE REVIEW

File: docs/versions/0.0.0.md [hunk 1]
```diff
@@ -180,6 +180,15 @@ Requirements
 
 6. Files in the `data` directory MUST NOT have filepaths that include the following characters: `/`, `:`, `*`, `?`, `"`, `<`, `>`, `|`, `;`, `#`.
 
+#### 2.7 `original-filepaths.txt`
+1. A DANS bag MAY contain a file `original-filepaths.txt` in the root of the bag
```
Comment (janvanmansum) [inline docs/versions/0.0.0.md:184]:
```suggestion
1. A DANS bag MAY contain a file `original-filepaths.txt` in the root of the bag.
```

File: docs/versions/0.0.0.md [hunk 2]
```diff
@@ -180,6 +180,15 @@ Requirements
...
+   
+2. It MUST be a UTF-8 encoded text file and MUST be formatted as `<physical-bag-relative-path><whitespace><orignal-bag-relative-path>`
```
Comment (janvanmansum) [inline docs/versions/0.0.0.md:186]:
```suggestion
2. It MUST be a UTF-8 encoded text file and MUST be formatted as `<physical-bag-relative-path><whitespace><orignal-bag-relative-path>`.
```

File: docs/versions/0.0.0.md [hunk 3]
```diff
@@ -180,6 +180,15 @@ Requirements
...
+   
+3. `<physical-bag-relative-path>` MUST NOT contain whitespace and MUST correspond one-on-one to an existing payload file
```
Comment (janvanmansum) [inline docs/versions/0.0.0.md:188]:
```suggestion
3. `<physical-bag-relative-path>` MUST NOT contain whitespace and MUST correspond one-on-one to an existing payload file.
```

File: docs/versions/0.0.0.md [hunk 4]
```diff
@@ -180,6 +180,15 @@ Requirements
...
+   
+4.  All `filepath` attributes in the `file` elements in the `files.xml` MUST appear in the `original-filepaths.txt` as an `<orignal-bag-relative-path>`
```
Comment (janvanmansum) [inline docs/versions/0.0.0.md:190]:
```suggestion
4.  All `filepath` attributes in the `file` elements in the `files.xml` MUST appear in the `original-filepaths.txt` as an `<orignal-bag-relative-path>`.
```

**Review by janvanmansum — CHANGES_REQUESTED** [review]
Dotted the proverbial i's, see comments. For the rest, looks OK to me.

File: docs/versions/0.0.0.md [hunk 5]
```diff
@@ -222,8 +230,9 @@ Requirements
 3. The document element MUST contain zero or more `file` elements, and MUST NOT contain other elements.
    <!-- Skip if XSD is used -->
 
-4. Each `file` element MUST have a `filepath` attribute which contains the bag local path to the payload file described.
-   Directories and non-payload files MUST NOT be described by a `file` element.
+4. Each `file` element MUST have a `filepath` attribute which contains the bag local path to the payload file described. 
+   When an `original-filepaths.txt` exists, rule 2.7.4 applies.
```
Comment (lindareijnhoudt) [inline docs/versions/0.0.0.md:234]:
```suggestion
   When an `original-filepaths.txt` exists, rule 2.7.2 applies.
```

**Pull request merged**